test/reduce.js

```diff
@@ -1,5 +1,6 @@
 var R = require('..');
 var eq = require('./shared/eq');
+var _xwrap = require('../src/internal/_xwrap');
```

Comment:
Uggh, I don't know if we do this anywhere else, but I really hate the idea of a test depending directly on our internals.